detox/src/client/actions/SyncStatusFormatter.test.js

```diff
@@ -0,0 +1,379 @@
+const format = require('./SyncStatusFormatter').formatJSONStatus;
+
+describe('Sync Status Formatter', () => {
```

Comment:
I'd suggest using snapshot tests in the majority of the cases here.

Reply:
done